src/site/stages/build/options.js

```diff
@@ -221,13 +222,77 @@ async function setUpFeatureFlags(options) {
   });
 }
 
+/**
+ * Sets up the CMS SideNav Menus list.
+ */
+async function setUpSideNavMenuList(options) {
+  global.buildtype = options.buildtype;
+  let SideNavs;
+
+  const sideNavFile = path.join(
+    options.cacheDirectory,
+    'drupal',
+    'side-nav-menus.json',
+  );
+
+  if (shouldPullDrupal(options)) {
+    logDrupal('Pulling side nav menus from Drupal...');
+    const apiClient = getDrupalClient(options);
+    const envConfig = DRUPALS[global.buildtype];
+    const drupalConfig = Object.assign({}, envConfig);
+    const { user, password } = drupalConfig;
+
+    const encodedCredentials = Buffer.from(`${user}:${password}`).toString(
+      'base64',
+    );
+    const result = await apiClient.proxyFetch(
+      `${apiClient.getSiteUri()}/graphql/`,
+      {
+        headers: {
+          Authorization: `Basic ${encodedCredentials}`,
+          'Content-Type': 'application/json',
+        },
+        method: 'post',
+        body: JSON.stringify({
+          query: `{
+         sideNavMenus
+       }`,
```

Comment:
I think it would make sense for this query/api logic to be a new method on `src/site/stages/build/drupal/api.js` like `getSideNavigations` or something